ethereum_history_api/circuits/get_receipt/src/main.nr

```diff
@@ -10,15 +10,15 @@ global MAX_LOG_DATA_SIZE = 128; // 4 * 32 (Eth word length)
 // MAX_RLP_LIST_HEADER_LENGTH = 3
 // MAX_RLP_ELEMENT_LENGTH = 32
 // LENGTH_OF_NULL_ELEMENT = 1
-// MAX_BRANCH_LENGTH = 3 + 16 * 32 + 1 = 515
-global MAX_RECEIPT_TRIE_NODE_LENGTH = 515;
-global MAX_RECEIPT_PROOF_LENGTH = 3090; // MAX_RECEIPT_TRIE_NODE_LENGTH * MAX_RECEIPT_TREE_DEPTH
+// MAX_BRANCH_LENGTH = 3 + 16 * 32 + 1 = 516
+global MAX_RECEIPT_TRIE_NODE_LENGTH = 516;
+global MAX_RECEIPT_PROOF_LENGTH = 3612; // MAX_RECEIPT_TRIE_NODE_LENGTH * MAX_RECEIPT_TREE_DEPTH
 global MAX_RECEIPT_RLP_LENGTH = 512; // TODO: Put the real value here
```

Comment:
MAX_RECEIPT_RLP_LENGTH needs to have the same value as MAX_RECEIPT_TRIE_NODE_LENGTH. Maybe let's change its value to 516 and add a comment explaining that the values need to be the same

Reply:
Food for thought: We should promote all constants to generics, for which it is hard to decide on the value.